connectors/http/http-base/src/main/java/io/camunda/connector/http/base/constants/Constants.java

```diff
@@ -27,7 +27,6 @@ public class Constants {
   public static final String CREDENTIALS_BODY = "credentialsBody";
   public static final String PROXY_FUNCTION_URL_ENV_NAME = "CAMUNDA_CONNECTOR_HTTP_PROXY_URL";
 
-  public static final String APPLICATION_JSON_CHARSET_UTF_8 = "application/json; charset=UTF-8";
   public static final String APPLICATION_X_WWW_FORM_URLENCODED =
       "application/x-www-form-urlencoded";
```

Comment:
This seems to be unused. Is that correct? 

I thought we now support this: https://github.com/camunda/connectors/pull/2291

Reply:
Deleted 👍

Reply:
Just curious, do we support sending data with this mime-type if its set via the header?

Reply:
[This](https://github.com/camunda/connectors/blob/397d35db8441b15d0716ac071bead43632b99071/connectors/http/http-base/src/test/java/io/camunda/connector/http/base/components/apache/CustomApacheHttpClientTest.java#L245) test should show this use case, is it correct?

Reply:
Yes, awesome. Thanks for the pointer.